OrangebeardLogger.cs

```diff
@@ -171,13 +190,16 @@ private void PopulateAttachmentData(ref string message, ref string attachmentMim
             }
 
 
+            var logLevel = DetermineLogLevel(level.Name);
             CreateLogItemRequest metaRq = null;
-            if ((int) DetermineLogLevel(level.Name) >= 3 && metaInfos.Count >= 1)
```

Comment:
The check if there is meta info no longer exists. Is that a deliberate change or an omission?

Reply:
This is an oversight. I'll fix this.

Reply:
Fixed. If you could check again, please?
Visual Studio also added a .csproj.pref and .sln.pref file. I'm not sure why, but they shouldn't harm. So I let them come along with the commit.